### ISSUE TITLE
[Installation] IPC_Path doesn't seem to work

### ISSUE THREAD

**Issue by georgeat8** [iamkroot/trakt-scrobbler#253]
### System Info
* Operating system and version: [eg: Windows 10, Linux Manjaro KDE] Windows 11
* Media Player and version: [eg: VLC v3.0.16, Plex v1.24.0.4930] SMPlayer 23.6.0
* Python version: [eg: 3.9] 3.11

### Description
After installing, scrobbler returns an IPC_PATH error (see logs below)

trakts_scrobbler config: 

![image](https://github.com/iamkroot/trakt-scrobbler/assets/62032417/9e84a8da-417f-47a3-a2c8-21f2d004cd76)

SMPlayer config: 

![image](https://github.com/iamkroot/trakt-scrobbler/assets/62032417/5f93635f-0850-4a59-a515-d7ae0db5622c)

### Log file

<details>
  <summary>Click to see log contents</summary>  

```
 2023-07-25 17:29:32,733 - INFO - smplayer@mpv - mpv - Unable to connect to MPV. Check ipc path.
 2023-07-25 17:30:53,780 - DEBUG - MainThread - notifier - Notifications enabled for categories: exception, misc, scrobble.pause, scrobble.resume, scrobble.start, scrobble.stop, trakt
 2023-07-25 17:30:53,940 - INFO - MainThread - scrobbler - Started scrobbler thread.
 2023-07-25 17:30:53,951 - DEBUG - MainThread - monitor - Autoloaded smplayer@mpv ipc_path = \\\\.\\pipe\\mpvsocket
 2023-07-25 17:30:53,951 - INFO - MainThread - monitor - Started monitor for smplayer@mpv
 2023-07-25 17:30:53,952 - INFO - smplayer@mpv - mpv - Unable to connect to MPV. Check ipc path.
```
</details>

**Comment by iamkroot** [comment]
> After installing, scrobbler returns an IPC_PATH error (see logs below)

Have you actually started playing any media in the player?

**Comment by iamkroot** [comment]
Hmm, it seems like there's some bug with the ipc path being autoloaded. We are escaping the slashes too much.
1. Could you please paste the contents of smplayer config- should be in `~/.smplayer/smplayer.ini` (`~` being your home directory, `C:\Users\username\`).
2. Try running `trakts config set players.smplayer@mpv.ipc_path '\\.\pipe\mpvsocket'` in a powershell window.

**Comment by iamkroot** [comment]
Let's try to simplify it even further. Can you connect to the smplayer ipc even without the scrobbler?
You can check by opening a powershell window and running `echo show-text ${playback-time} >\\.\pipe\mpvsocket`
(it should show the time in your smplayer window)

**Comment by iamkroot** [comment]
@georgeat8 Take a look at #266 - try https://github.com/iamkroot/trakt-scrobbler/issues/266#issuecomment-1873411091

**Comment by du33169** [comment]
The problem still exists for me (SMPlayer 25.6.0 on Windows 10), with the same log output.
I tried letting scrobbler autoload `ipc_path` from smplayer, setting `players.smplayer@mpv.ipc_path` or `players.mpv.ipc_path`, none of these worked.
I can control the player via command line.
I tried setting the `ipc_path` in smplayer to `\\.\pipe\\\.\pipe\mpvsocket` as mentioned in #266, still not working.
What did I miss?
